utils/dbdump/openmrs_small_sql

```diff
@@ -4637,7 +4637,7 @@ CREATE TABLE `patient` (
 
 LOCK TABLES `patient` WRITE;
 /*!40000 ALTER TABLE `patient` DISABLE KEYS */;
-INSERT INTO `patient` VALUES (7,2,'2021-01-14 12:22:56',NULL,NULL,0,NULL,NULL,NULL,'Unknown'),(8,2,'2021-01-14 12:23:01',NULL,NULL,0,NULL,NULL,NULL,'Unknown');
+INSERT INTO `patient` VALUES (7,2,'2021-01-14 12:22:56',NULL,NULL,0,NULL,NULL,NULL,'Unknown'),(8,2,'2021-01-14 12:23:01',NULL,NULL,0,NULL,NULL,NULL,'Unknown'),(9,2,'2021-03-10 12:23:01',NULL,NULL,0,NULL,NULL,NULL,'Unknown');
```

Comment:
Sorry, can you explain why you need to make changes to the test DB? I thought we just add a Patient and an Encounter/Observation for that Patient. Is this change to simplify ID extraction for the new patient?

Reply:
i made this change because i needed a patient without attached encounters or obs to use while doing the extraction because the current patients the demo data have many encounters and obs attached to them